### DISCUSSION TITLE
Conversion to/from Json

### DISCUSSION THREAD

**Discussion by cpotter1** [xlate/staedi#458] [General]
My use case is to translate x12 to Json, manipulate/transform in Json, and then translate back to x12. New to the space; can you suggest a framework/approach for this?

**Comment by MikeEdgar** [comment]
staedi supports several ways to read EDI data as JSON (see [`#createJsonParser`](https://javadoc.io/static/io.xlate/staedi/1.25.0/io.xlate.staedi/io/xlate/edi/stream/EDIInputFactory.html#createJsonParser(io.xlate.edi.stream.EDIStreamReader,java.lang.Class))), but there is currently no equivalent JSON writer.

**Reply by apoorva-sriv** [reply]
Can https://github.com/xlate/staedi/wiki/Reading-EDI-Data-(JSON-Conversion) be updated to show this new JSON parser?

**Reply by MikeEdgar** [reply]
@apoorva-sriv , yes that needs to be updated. Added to the to-do list :smile:
